ext/bin/pdbbox-init

```diff
@@ -36,20 +37,27 @@ while test $# -gt 0; do
             pgport="$1"
             shift
             ;;
+        --bind-addr)
+            shift
+            test ! -z $# || misuse
```

Comment:
Perhaps `test $# -gt 1`?

Reply:
the bash `-z` seems to be the canonical way to do this.